android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -145,6 +145,14 @@ public boolean onCreateOptionsMenu(Menu menu) {
         searchView.setQuery(searchText, false);
         return true;
     }
+    @Override
+    public void onBackPressed(){
+        if((mSessions.isEmpty())&&(sessionRecyclerView.getVisibility()==View.GONE)){
+            noSessionsView.setVisibility(View.VISIBLE);
```

Comment:
I don't think so. I tested it and it was working just fine. Could you confirm @mejariamol